lib/Models/MapInteractionMode.ts

```diff
@@ -12,8 +13,9 @@ interface Options {
   messageAsNode?: React.ReactNode;
   customUi?: () => unknown;
   buttonText?: string;
-  drawRectangle?: boolean;
+  // drawRectangle?: boolean;
```

Comment:
I have commented out `drawRectangle` as it refers to unimplemented (in mobx) drawing rectangle with SHIFT + drag